sorc/chgres_cube.fd/wam_climo_data.f90

```diff
@@ -0,0 +1,2667 @@
+        0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00,&
+        0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00, 3.84009E-02,&
+        3.96733E-02, 0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00,&
+        0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00, 5.05494E-02,&
+        7.39617E-02, 1.92200E-02,-8.46151E-03,-1.34244E-02, 1.96338E-02,&
+        1.50421E+00, 1.88368E+01, 0.00000E+00, 0.00000E+00,-5.13114E+01,&
+        0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00,&
+        5.11923E-02, 3.61225E-02, 0.00000E+00, 0.00000E+00, 0.00000E+00,&
+        0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00, 2.00000E+00/
+!          SEMIANNUAL MULT SAM                                          
+      DATA PAA1/                                                        &
+        1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00,&
+        1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00,&
+        1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00,&
+        1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00,&
+        1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00,&
+        1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00,&
+        1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00,&
+        1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00,&
+        1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00,&
+        1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00, 1.00000E+00/
+      DATA PAA2/                                                        &
+        0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00,&
+        0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00,&
+        0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00,&
+        0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00,&
+        0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00,&
+        0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00,&
+        0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00,&
+        0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00,&
+        0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00,&
+        0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00, 0.00000E+00/
+!         MIDDLE ATMOSPHERE AVERAGES                                    
+      DATA PAVGM/                                                       &
+        2.61000E+02, 2.64000E+02, 2.29000E+02, 2.17000E+02, 2.17000E+02,&
+        2.23000E+02, 2.86760E+02,-2.93940E+00, 2.50000E+00, 0.00000E+00/
+   END MODULE WAM_GTD7BK_MOD                                          
+! ---------------------------------------------------------------------
+   module gettemp_mod
+
+!x----use wam_gtd7bk_mod
+
+      REAL ::   TLB,S,DB04,DB16,DB28,DB32,DB40,DB48,DB01,ZA,T0,Z0,    &
```

Comment:
We need each module variable to be declared on it's own line, with a doygen comment attached. Like this:

```
REAL :: TLB !< What is TLB?
REAL :: S !< What is S?
etc.

```